.github/workflows/test.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - optimize-ci
```

Comment:
This will be removed again. It was only to trigger the CI run over at our fork.

-- https://github.com/crate-workbench/pycaret/actions/runs/6771777134